cgroup.go

```diff
@@ -48,11 +48,6 @@ func getAvailableCpusetList(cpusetReq string) (string, error) {
 
 	for k := range cpusetListReq {
 		if !cpusetGuestList[k] {
-			agentLog.WithFields(logrus.Fields{
```

Comment:
I wonder if rather than dropping completely, we could wrap them in a golang [once](https://golang.org/pkg/sync/#Once) ? But:
- maybe seeing it only once makes no sense
- we'd be adding a small overhead for the once checking
- you might have to pull the log code out to a new func so we can wrap it, which is just niggling.